index.js

```diff
@@ -39,10 +39,14 @@ async function terraform(args) {
 
 (async () => {
   const terraformDirectory = core.getInput('terraform_directory');
-  let terraformDoApply = core.getInput('terraform_do_apply') === 'true';
-  let terraformDoDestroy = core.getInput('terraform_do_destroy') === 'true';
-  const terraformLock = core.getInput('terraform_lock');
+  let terraformDoApply = core.getBooleanInput('terraform_do_apply');
+  let terraformDoDestroy = core.getBooleanInput('terraform_do_destroy');
+  const terraformLock = core.getBooleanInput('terraform_lock');
```

Comment:
Found this method by chance. It validates values against the YAML 1.2 "Core Schema" specification.
https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L161-L181